server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const port = 3000; 
```

Comment:
Consider using an environment variable for the port number instead of hardcoding it. This will make the application more flexible and easier to configure in different environments. [important]